### PR TITLE
Add recipe for bitsets

### PR DESCRIPTION
**Pull request by smirarab**
My first attempt to add bitsets here.

### CODE REVIEW

**Comment by smirarab** [comment]
@conda-forge-admin, please ping team

I am new to conda-forge. Adding a package for the first time. Any help with merging and maintaining would be much appreciated. The package is not mine. I need it for my tools, so adding it here.

File: recipes/bitsets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "bitsets" %}
+{% set version = "0.7.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/26/41/91d45f9818e25a0ce04a725a60639cdcf615772bb825cee8e8f0e706f8cd/bitsets-0.7.16.zip
```
Comment (synapticarbors) [inline recipes/bitsets/meta.yaml:9]: Please use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

You might have to change the `sha256`

File: recipes/bitsets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  imports:
+    - bitsets
+
+about:
+  home: "https://github.com/xflr6/bitsets"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.txt"
+  summary: "Ordered subsets over a predefined domain"
+  doc_url: ""
```
Comment (synapticarbors) [inline recipes/bitsets/meta.yaml:34]:
```
doc_url: https://bitsets.readthedocs.io/
dev_url: https://github.com/xflr6/bitsets
```

File: recipes/bitsets/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% set name = "bitsets" %}
+{% set version = "0.7.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/bitsets/bitsets-0.7.16.zip
```
Comment (synapticarbors) [inline recipes/bitsets/meta.yaml:9]: Could you use:

```
url: https://pypi.io/packages/source/b/bitsets/bitsets-{{ version }}.zip 
```

instead? I'm not sure if the hardcoded version will be handled properly by the regro-bot that automatically puts in a PR when the version is updated by the source.

**Pull request merged**